R/fread.R

```diff
@@ -28,7 +28,7 @@ yaml=FALSE, autostart=NA, tmpdir=tempdir(), tz="UTC")
     isTRUEorFALSE(stringsAsFactors) || (is.double(stringsAsFactors) && length(stringsAsFactors)==1L && 0.0<=stringsAsFactors && stringsAsFactors<=1.0),
     is.numeric(nrows), length(nrows)==1L
   )
-  fill=as.integer(fill)
+  fill = if(is.infinite(fill)) .Machine$integer.max else as.integer(fill)
```

Comment:
maybe identical(fill, Inf) instead? 
is.infinite(-Inf) is TRUE

Reply:
I had the same thought when writing it but thought if someone if willing to `fill=-Inf` then he deserves it :D

But yeah, should be unambigous in light of the mutation testing

Reply:
ok great thx